### ISSUE TITLE
data-transition="zoom" not working for vertical slides

### ISSUE THREAD

**Issue by bobaekang** [rstudio/revealjs#65]
I'm trying to implement `data-transition="zoom"` for a vertical slide (`##`) in 2-D presentation without much success. For example:

```md
---
title: "Presentation"
author: "Me"
date: "August 16, 2018"
output:
  revealjs::revealjs_presentation:
    center: true
    transition: concave
    reveal_options:
      slideNumber: true
---

## This works {data-transition="zoom"}
Zoom sucessful!

# Section

## This fails {data-transition="zoom"}
Zoom unsuccessful....
```

Please note that the first slide, although generated using `##`, is still horizontal. When it fails for the vertical slide, the `data-transition` seemingly defaults to `"slide"`.

Is there any fix or known workaround to this issue? Thank you in advance for your help!

**Comment by apreshill** [comment]
Can confirm that the second zoom transition does not work in the vertical slide with dev version of package. It does work using updated revealjs via Pandoc. Leaving open to test.

**Comment by cderv** [comment]
Yes I agree. It is possible than this has been fixed somehow in a reveal.js version then. 
I look for it but found an opened one: [data-transition on vertical sections hakimel/reveal.js#1974](https://github.com/hakimel/reveal.js/issues/1947)

So I don't know what too think. We'll need to see if this is fixed by a reveal.js update
